slither/core/solidity_types/array_type.py

```diff
@@ -32,6 +32,13 @@ def type(self):
     def length(self):
         return self._length
 
+    @property
```

Comment:
Is there a type that should not implement `storage_size`? If not I think we should extend `Type` from abc, and make it an `abc.abstractmethod`

It would be nice also to add types and an explanation why it returns a boolean